rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -608,6 +612,12 @@ def run(self):
         self.gcp_env.activate_sql_proxy()
         self.dao = GenomicSetMemberDao()
 
+        # Document these changes in the dev_note field
+        dev_note = input("Please enter a developer note for these updates:\n")
+        if dev_note in (None, ""):
+            _logger.error('A developer note is required for these changes.')
+            return 1
```

Comment:
Just sort of curious nit-picking: why use `input` here over a required CLI argument?

Reply:
My thought process was to ensure that when we update data, we are deliberately adding in a specific comment. It sort of serves as a reminder that we're manipulating data outside of the normal process. 
However, I do see use for it to be an argument too, as this would allow us to call it from other scripts in an automated fashion, though I don't like the idea of us having to update data like this very often. If it isn't too big an issue, I'd prefer to keep it as is for the time being. If it becomes cumbersome, we can update it in the future.

Reply:
Not a big issue at all, and after that I'm preferring that it stay the way it is. I agree that this should hopefully be used infrequently. At first I was thinking of it being used in other scripts, but this is a good way of making sure we're being careful. Otherwise it might be too easy to re-run as a stored command without paying attention to the note stored.